### PR TITLE
Small text updates

### PR DESCRIPTION
**Pull request by IAlibay**
A couple of small text updates based on what will probably be my near final read.

I'll add some justifications as comments.

### CODE REVIEW

File: papers/ian_kenney/ian_kenney.rst [hunk 1]
```diff
@@ -123,7 +123,7 @@ Centralized open-source packages such as MDAnalysis offer a limited solution
...
+One solution is to consolidate scientific code around a small number of large, central packages. MDAnalysis :cite:`michaud-agrawal_mdanalysis_2011, gowers_mdanalysis_2016` is a widely-used open-source Python library for molecular simulation data. With over 16 years of development by more than 180 developers, MDAnalysis has refined its code base to offer a mature, robust, flexible API that offers a range of high-performance tools to extract, manipulate, and analyze data from the majority of common simulation formats. MDAnalysis tools have been used for a variety of scientific applications ranging from exploring protein-ligand interactions :cite:`alibay_ialibaymdrestraintsgenerator_2021, kokh_workflow_2020, bouysset_prolif_2021`, to understanding lipid behavior :cite:`wilson_investigating_2021, smith_lipyphilic_2021`, to assessing the behavior of novel materials :cite:`gowers_kugupukugupu_2021, loche_maicos_2022`.
```
Comment (IAlibay) [inline papers/ian_kenney/ian_kenney.rst:126]: 18x developers now

File: papers/ian_kenney/ian_kenney.rst [hunk 2]
```diff
@@ -154,7 +154,7 @@ Our technical assistance begins with cookiecutter templates and example reposito
...
...ey are able to see new software as it gets added, rather than having to comb through literature or rely on developers advertising the code themselves. They are also able to easily verify the current development status of a package and whether it is being actively maintained and passing tests with both released and in-development versions of MDAnalysis. In the future, the registry could contain information about the health of a given codebase, such as its activity status and if it optimally leverages high performance MDAnalysis components (e.g. highly optimised PBC-aware distance routines). Packages on the registry also come with easy-to-find instructions on how to easily install and run a given package, significantly lowering the technical barrier to use and experimentation. As the maintenance remains the burden of the package owners, unfortunately the risk remains that packages on the registry may eventually become out-of-date, which is indeed one of the major disadvantages of this approach. However, the registry significantly increases the likelihood that packages will reach users who will become sufficiently motivated to contribute or take over their maintenance and development.
```
Comment (IAlibay) [inline papers/ian_kenney/ian_kenney.rst:157]: Here we say we're already checking if tests pass and then we say that in the future we'll check if the code has unit tests. I know that we probably meant "regression tests" vs "unit tests", but it felt more confusing than anything else. I'm happy to go with alternative wording here if there's any suggestions.

File: papers/ian_kenney/ian_kenney.rst [hunk 3]
```diff
@@ -329,7 +329,7 @@ MDAKit registry contents
...
+This metadata is used for two purposes: continuous integration testing and documentation. Continuous testing, helper methods and workflows are used to regularly install MDAKits and run their test suite (if available) to check if they still work as intended. Should the tests fail, package maintainers are automatically contacted and failure information is recorded in the code health metadata to inform users. For the registry documentation, the metadata is used to provide user-facing information about the various MDAKits in the registry, their contents, how to install them, and their current status as highlighted by continuous integration tests. The registry also includes further information such as; user guides and tutorials on the MDAKit framework, helping developers to implement their own MDAKits.
```
Comment (IAlibay) [inline papers/ian_kenney/ian_kenney.rst:332]: two ands on a sentence, maybe this is better?

File: papers/ian_kenney/ian_kenney.rst [hunk 4]
```diff
@@ -369,7 +369,7 @@ Once the criteria are fulfilled the metadata is merged and the MDAKit is conside
...
+Registered MDAKits are automatically added to the registry’s public facing documentation at https://mdakits.mdanalysis.org/mdakits.html. This involves an indexable list of entries for all registered MDAKits. Each entry displays available information from the provided metadata, e.g., what the MDAKit does, any relevant keywords, how to obtain the source code, how to install the package, and where to find relevant documentation. Alongside this information is also a set of badges which describe the current health of the codebase, allowing users to rapidly identify which packages are currently active, and their level of code maturity. This includes information such as which MDAnalysis library versions the package is compatible with. We further plan to add more infromation, such as how much test coverage the package has, and what type of MDAnalysis API extensions are provided (e.g., using base classes such as AnalysisBase or ReaderBase).
```
Comment (IAlibay) [inline papers/ian_kenney/ian_kenney.rst:372]: Again here we're saying "we currently runs all the tests and tell you if they fail" and then "in the future we'll tell you if the tests fail". I don't think we should include it in the "future" side of things.

**Review by orbeckst — APPROVED** [review]
lgtm

**Pull request merged**